src/components/EditUserMetadata.jsx

```diff
@@ -22,9 +22,13 @@ function getInitialFormValues(schema) {
   }, {});
 }
 
-const twitterSchema = find(intelligentAgentSchema, schema => {
-  return schema?.platformName === 'twitter'
-},{});
+const twitterSchema = find(
```

Comment:
I'm not sure find accepts a fallback argument actually! So this might not behave as expected. From the docs:
```
_.find(collection, [predicate=_.identity], [fromIndex=0])
```

Reply:
Good catch!